src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1575,14 +1575,14 @@ private void CheckModelAndSyntaxNodeToSpeculate(CSharpSyntaxNode syntax)
             {
                 Debug.Assert((object)container == null);
                 TypeSymbol containingType = binder.ContainingType;
-                TypeSymbol baseType = null;
 
                 // For a script class or a submission class base should have no members.
                 if ((object)containingType != null && containingType.Kind == SymbolKind.NamedType && ((NamedTypeSymbol)containingType).IsScriptClass)
                 {
                     return ImmutableArray<ISymbol>.Empty;
                 }
 
+                TypeSymbol baseType;
```

Comment:
Please revert all changes in this file.

Reply:
Uh, even this... I was told why you keep compiler layer unchanged above, but still I don't undrstand the philosophy behind it. Why just not to make a separate PR with only style changes and push it to all feature branches rather than keep so much redundant code there...

Reply:
It changes the history on these lines.